src/pages/Pool/index.tsx

```diff
@@ -145,7 +145,7 @@ export default function Pool() {
         {account && chainId && (
           <>
             <ButtonWithLink
-              link={`https://info.honeyswap.org/account/${account}?chainId=${chainId}`}
+              link={`https://info.honeyswap.org/account/${account}chainId=${chainId}`}
```

Comment:
This seems to be causing merge issues since it was modified in a previous commit. Make sure it matches the current version of the file